engine/runtime-parser-dsl/src/main/java/org/enso/runtime/parser/dsl/IRChild.java

```diff
@@ -0,0 +1,19 @@
+package org.enso.runtime.parser.dsl;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Constructor parameter annotated with this annotation will be represented as a child field in the
+ * generated super class. Children of IR elements for a tree. A child will be part of the methods
```

Comment:
```suggestion
 * generated super class. Children of IR elements form a tree. A child will be part of the methods
```

Reply:
Typo fixed in ac0d78bc84af28bab072cba5fb58d8e61caebe40